pvm.py

```diff
@@ -26,7 +26,7 @@ def _init_logger():
 _logger.info("Logging system initilized in %s", os.getcwd())
 
 # Place your videos in this folder for autostart
-PEFIX_PATH = "/home/pi/Videos/"
+PREFIX_PATH = "/home/pi/Videos/"
 VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
```

Comment:
As we discussed [here](https://github.com/omarcostahamido/PVM/pull/33#pullrequestreview-1178239338).
You maybe want to change this to `media = None`

Reply:
yes

Reply:
also please be aware that when you update this branch you will see another place where `media != ""` might need to be changed. believe it is line 81 as of https://github.com/omarcostahamido/PVM/commit/36a29c9c786b744b0bbeacb8118661c748150ea5

Reply:
Done, fix merge conflict.